packages/annotation-toolkit/src/layers/VideoPlayer.js

```diff
@@ -0,0 +1,128 @@
+import React, { useContext, useEffect, useCallback, useRef } from "react";
+import ReactPlayer from "react-player";
+import { useStore } from "global-context-store";
+import { Spinner } from "@blueprintjs/core";
+
+export default function VideoPlayer({
```

Comment:
Overall this looks great! I was curious how to communicate to the users of this component the API it provides. i.e. global data variables that are set ("playedSeconds", "duration", etc) and the actions that are supported (seek, ff, screenshot)

Reply:
Will add in a follow-up. Being tracked with #434